docs/troubleshoot_general_tips.rst

```diff
@@ -169,25 +169,38 @@ commands from within the repository::
 General git troubleshooting
 ---------------------------
 
-``git`` is powerful but complex; you may occasionally find your respository in a
+``git`` is powerful but complex; you may occasionally find your repository in a
 confusing state. This problem isn't devstack-specific.
 
-If you find yourself stuck, folks in the edX-internal or Open edX Slack workspaces may
-be able to give you a hand.
+If you find yourself stuck, folks in the 2U Slack workspace may be able to give
+you a hand.
+
+Ensure you are not still pointing to the old openedx repo. If you see openedx when running the following command::
+
+    % git remote -v
+    origin	ssh://git@github.com/openedx/devstack (fetch)
+    origin	ssh://git@github.com/openedx/devstack (push)
+
+You can point to the 2U fork of devstack with the following command::
+
+    % git remote set-url origin ssh://git@github.com/edx/devstack
```

Comment:
[inform] This is what I did locally, but my PR still got set up against openedx-unsupported at first, so there are other local changes I probably need to fix master, right?

Reply:
That's a separate thing we'll need to fix. GitHub knows that we forked edx/devstack from openedx-unsupported/devstack (either before or after the move from openedx) and so it assumes we want to contribute PRs to the upstream. I don't know if there's a way to change that behavior other than asking GitHub to unlink the repo from that "network".

Reply:
That's a bummer.